bin/pt-upgrade

```diff
@@ -6159,7 +6159,7 @@ sub _print_failures {
    foreach my $failure ( @$failures ) {
       print "\n-- $failno.\n";
       if ( ($failure->[1] || '') eq ($failure->[2] || '') ) {
-         printf "\nOn both hosts:\n\n" . ($failure->[1] || '') . "\n";
+         printf "\nOn both hosts:\n\n%s\n", ($failure->[1] || '');
```

Comment:
Just replacing the `printf` by a plain `print` would do the trick. 
The same in `lib/UpgradeResults.pm`

Reply:
That would be simpler. I modified the code according to what you requested.